ulib/experimental/Steel.Memory.fsti

```diff
@@ -51,8 +55,9 @@ val join_commutative (m0 m1:mem)
     (requires
       disjoint m0 m1)
     (ensures
-      (disjoint_sym m0 m1;
+      (disjoint m1 m0 /\
        join m0 m1 == join m1 m0))
+    [SMTPat (join m0 m1)]
```

Comment:
A bit afraid about this pattern triggering too often. Could you point me to proofs where this is required?

Reply:
In b2210d826868ee9b5925b26f7eecf429ae8172fc, I managed to remove this global pattern, instead using explicit calls in `Steel.Memory.dep_hprop_is_affine0` and a local pattern for the definition of the selector for vdep.